app/models/ability.rb

```diff
@@ -0,0 +1,27 @@
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    # Define abilities for the user here. For example:
+    user ||= User.new
+    if user.admin?
```

Comment:
- [ ] After removing that line in the model file, I ran into another error. You are calling an `admin?` method which you did not define anywhere in your User model. Kindly define the method.

  ![image](https://github.com/noelfoka/Blog_app/assets/54848363/17a845bf-253a-4435-bcc8-2aa4663be2d3)

- [ ] Kindly go through your app again and ensure that only admins and users who authored a post and comment can delete them.